src/components/navigation/README.md

```diff
@@ -0,0 +1,245 @@
+## Usage: Primary Components
+
+> These are the components that should be used in the **VAST MAJORITY** of use cases
+
+### d2l-navigation
+
+Add the `d2l-navigation` component, and provide sub elements `d2l-navigation-main-header` & `d2l-navigation-main-footer` (along with their respective slot contents).
+
+```html
+<script type="module">
+  import '@brightspace-ui/labs/navigation/d2l-navigation.js';
+  import '@brightspace-ui/labs/navigation/d2l-navigation-main-header.js';
+  import '@brightspace-ui/labs/navigation/d2l-navigation-main-footer.js';
+</script>
+
+<d2l-navigation>
+	<d2l-navigation-main-header>
+		<div slot="left" class="d2l-navigation-header-left">This should be on the left.  As the width changes it shrinks as needed.</div>
+
+		<div slot="right" class="d2l-navigation-header-right">This should be on the right.  It doesn't shrink.</div>
+	</d2l-navigation-main-header>
+	<d2l-navigation-main-footer>
+		<div slot="main" class="d2l-navigation-s-main-wrapper">Stuff goes in here (small border above and below)</div>
+	</d2l-navigation-main-footer>
+</d2l-navigation>
+```
+
+***Relevant CSS class name:***
+* `--d2l-navigation-shadow-drop-border-display`: The default value is `block`, but this property can be used to hide the shadow by setting it to `none`.
+
+### d2l-navigation-immersive
+
+Add the `d2l-navigation-immersive` component, providing values for the `backLinkHref` & `backLinkText`. Additionally, you may override any of the 3 slots (`left`, `middle`, `right`).
+Please note that overridding the `left` slot will prevent the Back link from displaying. This should only be done in very specialized cases.
+
+```html
+<script type="module">
+  import '@brightspace-ui/labs/navigation/d2l-navigation-immersive.js';
+</script>
+
+<d2l-navigation-immersive back-link-href="https://www.d2l.com" back-link-text="Back to D2L">
+	<div class="d2l-typography d2l-body-standard" slot="middle">
+		<p>Economics 101</p>
+	</div>
+	<div slot="right">
+		...
+	</div>
+</d2l-navigation-immersive>
+```
+
+Optionally:
+
+- The max-width can be configured to match the max-width used by the LE by setting `widthType` to `normal`
+- Overflow can be enabled to facilitate components like dropdowns by including the `allow-overflow` boolean attribute
+- A shorter version of the back text can be provided by setting the `back-link-text-short` attribute
+
+## Secondary Components
+
+> These are the components that make up the Primary Components. There might be an edge case or two where it makes sense to use one of these in isolation,
+> but **PLEASE STRONGLY CONSIDER** using a Primary Component instead.
+
+### d2l-navigation-band
+
+![screenshot of navigation band](/screenshots/navigation-band.png?raw=true)
```

Comment:
I don't think these screenshots made it over. I'd be OK with just removing them, since maintaining screenshots of components in READMEs isn't something we normally do.